app/helpers/checkout_helper.rb

```diff
@@ -152,4 +152,19 @@ def stripe_card_options(cards)
       ]
     end
   end
+
+  def link_to_service(baseurl, name, html_options = {}, &block)
+    return if name.blank?
+
+    html_options = html_options.merge target: '_blank'
+    link_to ext_url(baseurl, name), html_options, &block
+  end
+
+  def ext_url(prefix, url)
+    if url =~ %r{^https?://}i
+      url
+    else
+      prefix + url
+    end
+  end
```

Comment:
These are copies from the GroupsHelper. You could probably move them to the ApplicationHelper or create a new LinkHelper.

That said, the GroupsHelper contains nothing else except `strip_url` which seems to be unused. So you can remove that one and rename it to LinkHelper and everything should be fine.

I had trouble recently adding a new helper though. Somehow it wasn't loaded by Rails. I'm not sure what that was about. It may need some research if the helper isn't available straight away.